src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -47,6 +50,16 @@ export function generatePerformanceQuery(location: Location): Readonly<NewQuery>
         : '-rpm';
   }
 
+  if (query?.query) {
+    const searchQuery = decodeScalar(query.query);
+    if (searchQuery) {
+      extra.query = stringifyQueryObject({
+        query: [PERFORMANCE_EVENT_VIEW.query],
+        transaction: [`*${escapeDoubleQuotes(searchQuery)}*`],
```

Comment:
Shouldn't stringifyQueryObject be handling quotes?

Reply:
Now it does https://github.com/getsentry/sentry/pull/17763/commits/1557eac750db10715f6f6fec3e0b9ec3a3fa4f96